README.md

```diff
@@ -3,61 +3,50 @@
 The Grafana dashboard docker image is available at
 [Docker Hub](https://hub.docker.com/r/streamnative/apache-pulsar-grafana-dashboard).
 
-You have to provide following environment variables when using this dashboard image.
+To use this dashboard image, provide the following environment variables: 
 
-- *PULSAR_PROMETHEUS_URL*: The HTTP url that points to your prometheus service. E.g. `http://<prometheus-host>:9090`.
-- *PULSAR_CLUSTER*: The pulsar cluster name. The cluster name should be aligned with your prometheus configuration.
+- *PULSAR_PROMETHEUS_URL*: The HTTP URL that points to your prometheus service. For example, `http://<prometheus-host>:9090`.
+- *PULSAR_CLUSTER*: The pulsar cluster name. The cluster name is aligned with your prometheus configuration.
   See [Prometheus](#prometheus) for more details.
 
 ## Prometheus
 
-In order to make sure this dashboard can display the metrics correctly, you have to
-configure your Prometheus server to collect metrics from Pulsar correctly.
+To display the metrics correctly with this dashboard, configure your Prometheus server to collect metrics from Pulsar correctly.
 
-1. Make sure your prometheus service attach an extra label - `cluster`. The cluster name
-   should be aligned with the `PULSAR_CLUSTER` you provided to the grafana dashboard
+1. Attach your prometheus service to an extra label - `cluster`. The cluster name is aligned with the `PULSAR_CLUSTER` name you have provided to the grafana dashboard.
```

Comment:
> Attach your prometheus service to an extra label - `cluster`.

I am not sure if this is correct.  The original sentence is to describe - configure your prometheus service to attach an extra label - `cluster` to the metrics collected by prometheus.

Reply:
Sorry, I misunderstand the original sentence. Could we refine it as follows? 
"Configure Prometheus service, and make sure your Prometheus service attaches an extra label `cluster` to the metrics collected from Pulsar cluster."

Reply:
+1